src/main/flight/pid.c

```diff
@@ -865,28 +878,13 @@ FAST_CODE float featheredPids(const pidProfile_t *pidProfile, int axis, float er
 {
     (void)(currentPidSetpoint);
 
-//Add EmuBoost to the code (non linear boost to errorRate)
-float errorMultiplier = (pidProfile->errorBoost * pidProfile->errorBoost / 1000000) * 0.003;
-float boostedErrorRate = (errorRate * errorRate) * errorMultiplier;
-if (errorRate >= 0 && fabs(errorRate * pidProfile->errorBoostLimit / 100) > fabs(boostedErrorRate))
-  {
-    boostedErrorRate = (errorRate * errorRate) * errorMultiplier;
-  } else {
-    if ( errorRate < 0 && fabs(errorRate * pidProfile->errorBoostLimit / 100) > fabs(boostedErrorRate))
-  {
-    boostedErrorRate = (0 - errorRate * errorRate) * errorMultiplier;
-  } else {
-    boostedErrorRate = errorRate * pidProfile->errorBoostLimit / 100;
-  }
-}
-
     // -----calculate P component
-    pidData[axis].P = pidCoefficient[axis].Kp * (boostedErrorRate + errorRate);
+    pidData[axis].P = pidCoefficient[axis].Kp * errorRate;
```

Comment:
Does this change the error to always be a boost? I think that it is still good to have some linearity to the error. I could just be overlooking something here though.